app/src/main/java/com/peacecorps/pcsa/safety_tools/BystanderInterventionFragment.java

```diff
@@ -28,41 +25,42 @@
 public class BystanderInterventionFragment extends Fragment {
 
     public static final String TAG = BystanderInterventionFragment.class.getSimpleName();
-    Button potentialVictim,potentialOffender,tacticsForBoth;
+    Button potentialVictim, potentialOffender, tacticsForBoth;
     TextView safetyText;
+
     @Nullable
     @Override
     public View onCreateView(LayoutInflater inflater, @Nullable ViewGroup container, @Nullable Bundle savedInstanceState) {
 
-        View rootView =  inflater.inflate(R.layout.fragment_bystander,container,false);
+        View rootView = inflater.inflate(R.layout.fragment_bystander, container, false);
         potentialOffender = (Button) rootView.findViewById(R.id.potentialOffender);
         potentialVictim = (Button) rootView.findViewById(R.id.potentialVictim);
         tacticsForBoth = (Button) rootView.findViewById(R.id.tacticsForBoth);
         safetyText = (TextView) rootView.findViewById(R.id.bystander_text);
         safetyText.setText(Html.fromHtml(getActivity().getString(R.string.safety_text_bystander)));
-        safetyText.setTextSize(TypedValue.COMPLEX_UNIT_SP,26);
+        safetyText.setTextSize(TypedValue.COMPLEX_UNIT_SP, 26);
         safetyText.setGravity(Gravity.CENTER);
-        ((AppCompatActivity)getActivity()).getSupportActionBar().setTitle(R.string.bystander_intervention);
+        ((AppCompatActivity) getActivity()).getSupportActionBar().setTitle(R.string.bystander_intervention);
 
 
         potentialOffender.setOnClickListener(new View.OnClickListener() {
             @Override
             public void onClick(View v) {
-                SingleTextViewFragment.showSingleTextLayout(getActivity(),TAG,getString(R.string.bystander_potential_offender),getString(R.string.bystander_verbal_offender));
+                SingleTextViewFragment.showSingleTextLayout(getActivity(), getString(R.string.bystander_intervention), getString(R.string.bystander_potential_offender), getString(R.string.bystander_verbal_offender));
```

Comment:
used `getString(R.string.bystander_intervention)` instead `TAG` as fragment title in all the onClick listeners of potentialOffender/potentialVictim/tacticsForBoth.

Didn't change the string of TAG as it was not possible to fetch from strings.xml with it being static.